kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/TimeFieldParser.kt

```diff
@@ -20,6 +25,10 @@ interface TimeFieldParser {
                 Instant.from(timeFormatter.parse(line.getValue(fieldName)))
                         .toEpochMilli() / 1000.0
 
+        override fun timeFromString(timestamp: String): Double =
+                Instant.from(timeFormatter.parse(timestamp))
+                        .toEpochMilli() / 1000.0
+
```

Comment:
No need to override `time(line: Map<String, String>)` if you take the above suggestion.